packages/favorites-dialog/src/visTypes.js

```diff
@@ -17,20 +16,38 @@ import PivotTableIcon from './icons/PivotTableIcon';
 export const CHART = 'CHART';
 export const PIVOT_TABLE = 'PIVOT_TABLE';
 
-const visTypeMap = {
-    COLUMN: { label: i18n.t('Column'), icon: ColumnIcon },
-    STACKED_COLUMN: { label: i18n.t('Stacked column'), icon: StackedColumnIcon },
-    BAR: { label: i18n.t('Bar'), icon: BarIcon },
-    STACKED_BAR: { label: i18n.t('Stacked bar'), icon: StackedBarIcon },
-    LINE: { label: i18n.t('Line'), icon: LineIcon },
-    AREA: { label: i18n.t('Area'), icon: AreaIcon },
-    PIE: { label: i18n.t('Pie'), icon: PieIcon },
-    RADAR: { label: i18n.t('Radar'), icon: RadarIcon },
-    GAUGE: { label: i18n.t('Gauge'), icon: GaugeIcon },
-    YEAR_OVER_YEAR_LINE: { label: i18n.t('Year over year (line)'), icon: YearOverYearLineIcon },
-    YEAR_OVER_YEAR_COLUMN: { label: i18n.t('Year over year (column)'), icon: YearOverYearColumnIcon },
-    SINGLE_VALUE: { label: i18n.t('Single value'), icon: SingleValueIcon },
-    [PIVOT_TABLE]: { label: i18n.t('Pivot table'), icon: PivotTableIcon },
+export const visTypeIcons = {
```

Comment:
Split the icons and labels, since labels has to be gotten from a function (to ensure string translation)